qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestBase.java

```diff
@@ -421,7 +350,7 @@ public String createTestLocationTag() {
 	 * @return The Patient Identifier for the newly created patient
 	 */
 	public String createPatient(String personUuid,
-			String patientIdentifierType, String source) {
+								String patientIdentifierType, String source) {
```

Comment:
any reason for this change?

Reply:
i think my IDE its the one padding it @kdaud

Reply:
@kdaud @sherrif10  please re-run the workflow for the failing 2.x

Reply:
so are you fixing the issue?

Reply:
yes @kdaud